src/components/gataca-qr/gataca-qr.tsx

```diff
@@ -422,6 +423,7 @@ export class GatacaQR {
           resolve(data);
           break;
         case RESULT_STATUS.ONGOING:
+        case RESULT_STATUS.READ:
```

Comment:
@jose-gataca I'm trying the new version of the QR in studio ([PR for tests](https://github.com/gataca-io/studio/pull/264)), and can't get the RESULT_STATUS.READ, and I think that when we added it in here isn't changing between cases, because the QR doesn't differentiates between status